### PR TITLE
Document use of CND for namespaces

### PR DESCRIPTION
**Pull request by younnoh**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -59,3 +59,9 @@ optional arguments:
                         ERROR)
   -v, --verbose         Show detailed info for each resource checked
 ```
+
+## Namespaces
```
Comment (bseeger) [inline README.md]: Hi @younnoh - thanks for writing this.  I think this should be on the import-export tool (https://github.com/fcrepo4-labs/fcrepo-import-export) instead, not the verify tool

Comment (younnoh) [inline README.md]: Thanks, @bseeger, for this and for your comments on the ticket.

Comment (awoods) [inline README.md]: @bseeger : agreed

**Comment by younnoh** [comment]
Resubmitted as https://github.com/fcrepo4-labs/fcrepo-import-export/pull/83

**Pull request closed without merging**